docker-run.sh

```diff
@@ -26,17 +30,26 @@ OUTPUT_CODED_DIR=${12}
 OUTPUT_MESSAGES_CSV=${13}
 OUTPUT_INDIVIDUALS_CSV=${14}
 OUTPUT_PRODUCTION_CSV=${15}
+DRIVE_AUTH_FILE=${16}
+MESSAGES_DRIVE_PATH=${17}
+INDIVIDUALS_DRIVE_PATH=${18}
+PRODUCTION_DRIVE_PATH=${19}
 
 # Build an image for this pipeline stage.
 docker build -t "$IMAGE_NAME" .
 
-# Create a container from the image that was just built.
-CMD="pipenv run python -u redss_pipeline.py $USER /data/phone-number-uuid-table-input.json
-    /data/s01e01-input.json /data/s01e02-input.json /data/s01e03-input.json /data/s01e04-input.json
-    /data/demog-input.json /data/evaluation-input.json /data/prev-coded
-    /data/output.json /data/output-icr /data/coded
-    /data/output-messages.csv /data/output-individuals.csv /data/output-production.csv"
-container="$(docker container create -w /app "$IMAGE_NAME" ${CMD})"
+# Create a container from the image that was just built, making gcloud credentials available via a volume mount.
+CMD="
+    gsutil cp $DRIVE_AUTH_FILE /.config/drive-service-account-credentials.json && \
```

Comment:
why is this a gsutil cp? Could it be a plain cp instead? Unless I'm seriously misunderstanding something it's doing a local -> local copy, rather than working with gs:// urls.